### ISSUE TITLE
Follower to run other areas after Team Baal

### ISSUE THREAD

**Issue by N33dl3s** [kolton/d2bot-with-kolbot#1864]
So I have a Paladin and a Sorc running Diablo and Baal using D2BotLead/D2BotFollow and the DiabloHelper and BaalHelper scripts. I have set the bosses at the beginning of the run, and after Baal is dead I'd like to do a few other areas, each of them seperately.
For the leader, this is not a problem and works perfectly fine when I put the scripts in the order I want, but the leecher always does the solo scripts first, and then the Dia / Baal scripts.

Is there a way to solve this? Do I have to run them both as Leaders, and if so, how would I set them up to do the same games. 

I've used the search function and have not found an answer.

**Comment by Jestah66** [comment]
It's all in the timings,  my sorc starts the game - does the pit, countess, meph, eldritch, shenk, dac..... while my follower strafezon does pindle and then waits for the throne TP for a few mins.  You just need to play with the timings till they are happy little bots.  Runtimes for the cycle are about 6 - 8 mins depending.....

Config.BaalHelper.Wait = 530; // Seconds to wait for a runner to be in Throne

**Comment by N33dl3s** [comment]
Right, so what you would suggest ist do all the other runs beforehand and then afterwards start the Diablo/Baal teamrun. I'll try to play around with it and report back how it works.
Maybe something like:

Sorc -> Mausoleum, Andy, Ancients, Meph
Pala -> Countess, Pits, Nihla 

Do you know if the follower will just idle after he is finished, and take the TP once it's up?

In any case: thanks for the help. I'll try it out and report back.

**Comment by Jestah66** [comment]
No problem.  That's it exactly.   My follower waits, however I'm only running baal helper to team the big guy and use the timeout in the char config.   Bringing up a baby smiter now, so I'll have to adjust my runs for him.  You might have to customize/tweak the follower runs so he ends up ready for the two bigger bosses when the leader is.
